# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../sge/fix.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- gridengine/source/dist/util/arch.fixed	2005-10-19 21:07:47.000000000 +0200
+++ gridengine/source/dist/util/arch	2005-10-19 21:07:58.000000000 +0200
@@ -45,9 +45,9 @@
 
 PATH=/bin:/usr/bin:/usr/sbin
 
-ARCH=UNKNOWN
+ARCH=glinux
 
-if [ -x /bin/uname ]; then
+if [ -x /bin/unameX ]; then
    os="`/bin/uname -s`"
    ht="`/bin/uname -m`"
    osht="$os,$ht"
